_posts/2021-03-10-katib-0.10.md

```diff
@@ -0,0 +1,310 @@
+[Kubernetes workloads](https://kubernetes.io/docs/concepts/workloads/) and
+Kubernetes [custom resource definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+to perform Katib Experiments. Since Katib can execute various Kubernetes resources,
+users are able to run Experiments which can optimize not only ML models but also
+any software, code or program to make it more efficient with optimization
+algorithms provided by Katib.
+
+Moreover, we are continually working on the new Katib UI to perform better User
+Experience and native integration with Kubeflow central dashboard.
+Please check [this presentation](https://youtu.be/OKqx3IS2_G4?list=PLmzRWLV1CK_ypvsQu10SGRmhf2S7mbYL5)
+to know more about the new UI.
+
+All of the above mentioned features allow users to easily integrate Katib in
+their ML infrastructure pipeline.
+
+## System Architecture
+
+<img src="/images/2021-03-10-katib-0.10/architecture.png" width="" alt="Architecture" title="">
+
+There are 3 main concepts in Katib which are Kubernetes CRDs:
+
+- **Experiment** - a single optimization run with objective, search space, and
+  search algorithm.
+
+- **Suggestion** - set of hyperparameters, which are produced by a user's
+  selected search algorithm. Katib creates Trials to evaluate them.
+
+- **Trial** - one iteration of the hyperparameters tuning process. Trial runs
+  the worker job which corresponds to the training job. Since Trial is an
+  abstraction on top of the worker job, any Kubernetes resource can be used to
+  perform the training job. For example, [**TFJob**](https://github.com/kubeflow/tf-operator),
+  [**MPIJob**](https://github.com/kubeflow/mpi-operator) or even
+  [Tekton **Pipeline**](https://github.com/tektoncd/pipeline).
+
+By using above resources Katib follows the following steps, which are marked
+in the diagram above:
+
+1. Once an Experiment is submitted, the Experiment controller creates an
+   appropriate Suggestion object.
+
+1. The Suggestion controller creates an AutoML algorithm service based on this
+   Suggestion object. When the algorithm service is ready, the Suggestion controller
+   calls the service to get new parameters and appends them to the Suggestion object.
+
+1. The Experiment controller finds that Suggestion object has been updated and
+   creates a corresponding Trial object for each set of parameters.
+
+1. The Trial controller generates a worker job for each Trial object and watches
+   for the status of each job. The worker job based on the Trial template.
+
+1. Once the worker job has been completed, the metrics collector gets the metrics
+   from the job and persists them in the database.
+
+1. The Experiment controller sends the metrics results to the algorithm service
+   and gets new parameters from the Suggestion object.
+
+## Custom Kubernetes resources support
+
+The latest Katib version 0.10 implements a new feature to support any Kubernetes
```

Comment:
Revise sentence to start with "Katib version 0.10" or rephrase to "The lates Katib version, 0.10, implements..."